ParkingLot.java

```diff
@@ -55,12 +55,12 @@ public int getTotalParkingSpaces() {
 
     /**
      *Parks at the requested slot if it is empty and prints a message
-     *indicating you have parked successfully, else throws exception
+     *indicating you have parked successfully, else throws exception for an invalid value
      *@param parkingSpotToPark is the parking spot that they are trying to park at
      **/
     public void park(int parkingSpotToPark) throws Exception {
```

Comment:
same comment as above. use throws to specify the particular type of exception thrown. later if you have more than one exception thrown, you can say "throws <exceptiontype1>, <exceptiontype2>..."